.github/workflows/k8s_matrix_test.yaml

```diff
@@ -0,0 +1,14 @@
+# Runs the single-cluster e2e tests with various versions of Kubernetes, to ensure backward compatibility.
+name: K8s version test
+on:
+  schedule:
+    - cron: '0 12 * * 0' # Every Sunday at noon UTC
+jobs:
+  run-e2e-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        kind_node_version: ["v1.22.15", "v1.23.6", "v1.24.7"]
+    uses: ./.github/workflows/kind_e2e_tests.yaml
+    with:
+      kind_node_version: ${{ matrix.kind_node_version }}
```

Comment:
No need to use `benc-uk/workflow-dispatch` anymore, there is now a built-in way to do this.

Reply:
Awesome, it's indeed much better this way as all the triggered job are in the same actions run 👏